_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -22982,8 +22971,8 @@
 /turf/open/floor/iron/dark,
 /area/station/command/bridge)
 "hhf" = (
-/obj/machinery/power/smes{
-	charge = 5e+06
+/obj/machinery/power/smes/full{
+	output_level = 30000
 	},
```

Comment:
should we make this a subtype in the code so coders can more readily be aware of its existence when we inevitably fuck with how SMES's/power work in the future

Reply:
Why even use 30000? What's wrong with the default 50000? Just remove this var edit altogether

Reply:
that's true too